napari/utils/colormaps/categorical_colormap.py

```diff
@@ -60,7 +60,9 @@ def map(self, color_properties: Union[list, np.ndarray]) -> np.ndarray:
             ]
             for prop in props_to_add:
                 new_color = next(self.fallback_color.cycle)
-                self.colormap[prop] = np.squeeze(transform_color(new_color))
+                self.colormap[prop] = ColorValue(
+                    np.squeeze(transform_color(new_color))
+                )
```

Comment:
ColorValue already gets coerced on initialization (with `transform_color`), so I think we can simplify this to:

```suggestion
                self.colormap[prop] = ColorValue(new_color)
```